lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3325,6 +3325,26 @@ namespace {
       return result;
     }
 
+    bool isCxxRecordUnimportable(const clang::CXXRecordDecl *decl) {
```

Comment:
Nit: I think it would be preferable to name without the negative `isCxxRecordImportable` and use `!` at existing call-sites. Otherwise, if some code ends up having `!isCxxRecordUnimportable` the double-negative could get confusing.